wiki/Ranking_Criteria/osu!taiko/es.md

```diff
@@ -1,46 +1,163 @@
-osu!taiko Ranking Criteria
-=====================================
+osu!taiko 
```

Comment:
Esto deberia ser `# osu!taiko`, ya no usamos "===" para los titulos.

Reply:
Me podrías explicar un poco a que te refieres? lo que entiendo yo es que hay que cambiar la linea de "osu!taiko" por "#osu!taiko" y borrar la linea de abajo que tiene los "=============". Correcto?

Reply:
Yeah.

Reply:
clayton will fix that in a moment >~<

Reply:
done